lib/core/test/unit/Cardano/Wallet/CoinSelection/Internal/BalanceSpec.hs

```diff
@@ -1203,20 +1215,28 @@ prop_performSelectionEmpty mockConstraints (Small params) =
         else
             -- If the initial list of outputs is non-empty, then no
             -- transformation should take place:
-            [ params === (paramsTransformed & over #outputsToCover F.toList)
-            , resultTransformed === (result & over #outputsCovered F.toList)
+            [ params === paramsTransformed'
+            , resultTransformed === result'
```

Comment:
Similar to above, this section of code is changed to use ordinary record update syntax instead of `over` (from `generic-lens`).